distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/MachineDependentHeap.java

```diff
@@ -37,6 +37,7 @@ public class MachineDependentHeap {
     protected static final long GB = 1024L * 1024L * 1024L; // 1GB
     protected static final long MAX_HEAP_SIZE = GB * 31; // 31GB
     protected static final long MIN_HEAP_SIZE = 1024 * 1024 * 128; // 128MB
+    protected static final long SERVER_CLI_OVERHEAD = 120 * 1024L * 1024L;
```

Comment:
I'd make this an even 100mb for now. I don't think we need _this_ much cushion. We aren't consistently breaking the limit as it is.